### ISSUE TITLE
Parser doesn't handle BOM in SRT file

### ISSUE THREAD

**Issue by sampattuzzi** [glut23/webvtt-py#5]
I noticed that trying to parse and SRT file I would get the following error:

```
----> 1 WebVTT().from_srt('5_IS_URC.srt')

C:\Users\sam\scoop\apps\python\3.5.2\lib\site-packages\webvtt\main.py in f(self, file)
     50         def f(self, file):
     51             self.file = file
---> 52             self._captions = parser_class().read(file).captions
     53             return self
     54

C:\Users\sam\scoop\apps\python\3.5.2\lib\site-packages\webvtt\generic.py in read(self, file)
    116
    117         content = self._read_content(file)
--> 118         self._validate(content)
    119         self._parse(content)
    120

C:\Users\sam\scoop\apps\python\3.5.2\lib\site-packages\webvtt\parsers.py in _validate(self, lines)
     86     def _validate(self, lines):
     87         if len(lines) < 2 or lines[0] != '1' or not self._validate_timeframe_line(lines[1]):
---> 88             raise MalformedFileError('The file does not have a valid format.')
     89
     90     def _is_timeframe_line(self, line):

MalformedFileError: The file does not have a valid format.
```

On further inspection, this is because my file begins with a BOM:
```
    def _validate(self, lines):
        if len(lines) < 2 or lines[0] != '1' or not self._validate_timeframe_line(lines[1]):
            import pdb; pdb.set_trace()
            raise MalformedFileError('The file does not have a valid format.')
```
```
(Pdb) lines[0]
'\ufeff1'
(Pdb) lines[0] == '1'
False
```

This could be fixed by stripping the BOM.

**Comment by sampattuzzi** [comment]
My fix is here: https://github.com/glut23/webvtt-py/pull/8

**Comment by glut23** [comment]
Hi @sampattuzzi this issue should be fixed with your updates, could you include a test case to cover parsing a caption file with BOM? Thanks!

**Comment by glut23** [comment]
Hi @sampattuzzi I released version 0.4.0 that handles captions with BOM and includes a test case. If you can confirm it works for you and close the issue if so? Thanks!

**Comment by glut23** [comment]
Closing this issue as BOM is now supported.
